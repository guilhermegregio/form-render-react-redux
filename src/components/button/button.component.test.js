import React from 'react';
import ReactDOM from 'react-dom';

import Button from './button.component';

it('default render button component', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Button id="1" value="Gravar" className="button" />, div);

    expect(div).toMatchSnapshot();
});
