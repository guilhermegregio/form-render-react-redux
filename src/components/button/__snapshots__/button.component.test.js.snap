// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render button component 1`] = `
<div>
  <div
    data-reactroot=""
    class="button-component group button"
  >
    <button
      id="1"
      type="button"
      class="button primary"
      value="Gravar"
    >
      <span
        class="load fa fa-spinner"
      />
      <span
        class="value"
      >
        Gravar
      </span>
    </button>
  </div>
</div>
`;
